src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java

```diff
@@ -6,20 +6,17 @@
 
     public final String envelopeId;
     public final String jurisdiction;
-    public final String service;
     public final String exceptionRecordRef;
     public final String newCaseRef;
 
     public UpdatePaymentMessage(
-        @JsonProperty(value = "envelope_id", required = true) String envelopeId,
-        @JsonProperty(value = "jurisdiction", required = true) String jurisdiction,
-        @JsonProperty(value = "service", required = true) String service,
+        @JsonProperty(value = "envelope_id") String envelopeId,
+        @JsonProperty(value = "jurisdiction") String jurisdiction,
```

Comment:
we just need exception_record_ref and new_case_ref to update payment in payhub. 
thought better to be flexible for  update.
if jurisdiction and envelope id is not there still we should update?

Reply:
but do you expect orchestrator to not send it in some scenarios and in some scenarios it will send. Contract is between what orchestrator sends to payments queue and what payment processor parses. what is being sent to pay hub can be completely different.

Reply:
reverted

Reply:
Should we prevent updating payment info if exception record has no `envelope_id` on it for some reason @nitinprabhu ?